playbooks/roles/hypershift/defaults/main.yml

```diff
@@ -0,0 +1,21 @@
+# hypershift default variables
+hypershift_install: false
+hypershift_destroy_cluster: false
+mgmt_cluster_kubeconfig: "{{ ansible_env.HOME }}/.kube/config"
+hosted_cluster_name: "rdr-hypershift-cluster"
+hypershift_dir: "/root/hypershift"
+hypershift_git_repo: "https://github.com/openshift/hypershift"
+hypershift_git_branch: "main"
+ibmcloud_resource_group: ""
+powervs_region: ""
+powervs_zone: ""
+vpc_region: ""
+base_domain: ""
+nodepool_replicas: 2
+ocp_pull_secret_path: "~/.pullSecret"
+sys_type: "s922"
+proc_type: "dedicated"
+processors: "0.5"
+memory: "32"
+infra_id: "rdr-hypershift-infra"
+ocp_release_image: ""
```

Comment:
Again, the default empty value will fail the create command. Better to remove it from here.